dags/first_shutdown_summary.py

```diff
@@ -0,0 +1,34 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from operators.emr_spark_operator import EMRSparkOperator
+from utils.tbv import tbv_envvar
+
+default_args = {
+    'owner': 'amiyaguchi@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2018, 3, 29),
+    'email': ['telemetry-alerts@mozilla.com', 'amiyaguchi@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=30),
+}
+
+# Make sure all the data for the given day has arrived before running.
+# Running at 1am should suffice.
+dag = DAG('first_shutdown_summary', default_args=default_args, schedule_interval='0 1 * * *')
+
+first_shutdown_summary = EMRSparkOperator(
+    task_id="first_shutdown_summary",
+    job_name="First Shutdown Summary View",
+    execution_timeout=timedelta(hours=14),
```

Comment:
Does this actually take 14 hours?

Reply:
Ah no, this should be less than an hour.